lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -278,6 +278,16 @@ class UsdUndoAddNewPrimAndSelectCommand : public Ufe::CompositeUndoableCommand
                 Ufe::GlobalSelection::get(), newSelection));
         }
     }
+
+#ifdef UFE_V4_FEATURES_AVAILABLE
+#if (UFE_PREVIEW_VERSION_NUM >= 4032)
```

Comment:
Should this be 4034 to match your Ufe PR?

Reply:
No the change is coming from a previous PR. Refer to https://git.autodesk.com/media-and-entertainment/ufe/pull/302 for details.